lib/Mojolicious/Guides/Cookbook.pod

```diff
@@ -342,6 +342,51 @@ web features, such as WebSockets, are not available.
 
   ScriptAlias / /home/sri/my_app/script/my_app/
 
+=head2 Envoy Proxy
+
+For deploying L<Mojolicious> applications on "cloud-native" environments that use L<Envoy|https://www.envoyproxy.io>,
+here is a configuration for a basic reverse proxy similar to Apache and Nginx:
```

Comment:
Other sections don't use the "...:" format for examples.

Reply:
Yeah, I'll need to rephrase this :muscle: